modules/ROOT/pages/gov-cloud-features.adoc

```diff
@@ -6,6 +6,13 @@ endif::[]
 MuleSoft Government Cloud includes the full functionality of the Anypoint Platform runtime plane (services and Mule runtime engine) and a subset of the Anypoint Platform control plane (browser-based management of APIs and Mule runtime engine instances) functionality. Government Cloud supports the full API-led connectivity methodology and end-to-end API lifecycle management.
 
 MuleSoft Government Cloud supports Mule runtime engine v3.9 and later.
+[IMPORTANT]
+====
+Because MuleSoft Government Cloud does not support Mule agents with self-signed certificates, specific versions of Mule runtime engine are required:
+
+* Mule 3: v1.12.0 and later
+* Mule 4: v2.4.9 and later
```

Comment:
Mule 4: v2.2.0 and later

v2.2.0 -> Public Certificates support
v2.4.9 -> Fips support